doc/devguide/codebase/unittests-c.rst

```diff
@@ -1,12 +1,152 @@
-Unittests
-=========
+**************
+Unit Tests - C
+**************
 
-Unittests are a great way to create tests that can check the internal state
+Unit tests are a great way to create tests that can check the internal state
 of parsers, structures and other objects.
 
 Tests should:
 
-- use FAIL/PASS macros
+- use ``FAIL``/``PASS`` macros
 - be deterministic
-- not leak memory on PASS
+- not leak memory on ``PASS``
 - not use conditions
+
+Unit tests are used by developers of Suricata and advanced users who would like to contribute by debugging and testing the engine.
+Unit tests are small pieces (units) of code which check certain code functionalities in Suricata. If Suricata's code is modified, developers can run unit tests to see if there are any unforeseen effects on other parts of the engine's code.
+Unit tests will not be compiled with Suricata by default.
+If you would like to compile Suricata with unit tests, enter the following during the configure-stage::
+
+   ./configure --enable-unittests
+
+The unit tests specific command line options can be found at `Command Line Options <https://suricata.readthedocs.io/en/suricata-6.0.3/command-line-options.html#unit-tests>`_.
+
+Example:
+You can run tests specifically on flowbits. This is how you should do that::
+
+   suricata -u -U flowbit
+
+It is highly appreciated if you would run unit tests and report failing tests at Issues.
```

Comment:
nit: Issues could probably link to Redmine?

Reply:
Oh, it should be linking. Thanks, this change must have slipped at some point >___<